selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -507,17 +525,35 @@ public void setCursorToLine(int positionLine) {
     loader.waitOnClosed();
     askForValueDialog.clickOkBtn();
     askForValueDialog.waitFormToClose();
-    waitActiveEditor();
+    waitActive();
     expectedNumberOfActiveLine(positionLine);
   }
 
   /**
-   * set cursor in specified position
+   * set cursor in specified position for current visible editor
    *
    * @param positionLine is the specified number line
    * @param positionChar is the specified number char
    */
-  public void setCursorToDefinedLineAndChar(int positionLine, int positionChar) {
+  public void setCursorToDefinedLineAndCharForVisible(int positionLine, int positionChar) {
```

Comment:
Could you, please, explain, what is the difference between the **current**, **visible** and **active** state of editor?

Reply:
For instance: we can have some visible cursor positions in case split editor. But if editor is not split - visible cursor widget can be just one. Active editor - it means that editor tab is selected and cursor has been set. Visible != active. Because cursor position widget can be visible  in not active editor. In some test enough just visible value - in another cases we should check active state an editor.

Reply:
Thank you for explanation.
If I understood it correctly _Current_ is always _Active_ and _Visible_. Therefore, **CurrentActive** could be shorten to **Current**.